.github/workflows/e2e-tests-wp-4-7.yml

```diff
@@ -65,10 +71,7 @@ jobs:
           restore-keys: ${{ runner.os }}-composer-
 
       - name: Composer Install
-        uses: php-actions/composer@v4
-        with:
-          dev: no
-          php_version: 7.3
+        run: composer install --no-interaction --no-progress
```

Comment:
@aaemnnosttv I think we can also use the `--no-dev` flag since we don't use development dependencies in the e2e tests, right? Let's add it then.

Reply:
Oops yeah, you're right 👍  Will fix.